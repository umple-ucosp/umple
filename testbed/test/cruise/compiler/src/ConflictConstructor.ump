generate Java "../src-gen-umple";
class A
{
	A (D d) {} 
}

class D
{

	1 -- * A a;
}

