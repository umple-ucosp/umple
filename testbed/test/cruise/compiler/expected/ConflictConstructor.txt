Warning 170 on line 4 of file "ConflictConstructor.ump":
Custom constructors should normally not be provided in Umple. Consider using before or after keywords to adjust what the constructor does, or adjusting constructor parameters using keywords like lazy.
Error 171 on line 4 of file "ConflictConstructor.ump":
A custom constructor should not conflict with an auto-generated constructor.
